.github/workflows/fossa.yaml

```diff
@@ -0,0 +1,32 @@
+name: Fossa scan
+
+on:
+    push: 
+      branches:
+        - '*'
+      tags:
+        - '*'
+    pull_request:
+      branches:
+        - '*'
```

Comment:
Do not run the workflow on pull requests, because the FOSSA token might be exposed to unauthorized external contributors that open PRs in the repo.

Reply:
Updated, thanks for pointing that out.